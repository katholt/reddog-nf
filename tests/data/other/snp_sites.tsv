#CHROM	#POS
contig_1	1158
contig_1	1644
contig_1	2081
contig_1	2086
contig_1	2092
contig_1	4807
contig_1	4818
contig_1	4820
contig_1	4850
contig_1	6268
contig_1	6473
contig_1	6484
contig_1	6485
contig_1	6506
contig_1	6525
contig_1	8001
contig_1	10954
contig_1	10955
contig_1	12053
contig_1	13373
contig_1	13377
contig_1	13392
contig_1	14636
contig_2	157
contig_2	560
contig_2	561
contig_2	779
