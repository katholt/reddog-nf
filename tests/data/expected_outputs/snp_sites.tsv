../other/snp_sites.tsv